File: Kata FizzBuzz/fizzBuzz.js
// for (var number = 1; number <=100; number++)

function fizzBuzz (number) {

        if (number % 3 == 0 && number % 5 == 0) {
            return "FizzBuzz";
        }
        
        else if (number % 5 == 0) {
            return "Buzz";
        }

        else if (number % 3 == 0) {
            return "Fizz";
        }
        
        else {
            return number; 
        }
    }

module.exports = fizzBuzz;

File: FizzBuzz&Test Java/src/main/java/FizzBuzz.java
public class FizzBuzz {

    public static String getResult(int number){
            if (number % 5 == 0 && number % 3 == 0) {
                return "fizzbuzz";
            } else if (number % 3 == 0) {
                return "fizz";
            } else if (number % 5 == 0) {
                return "buzz";
            }
                return Integer.toString(number);
    }
}


